src/views/CardStack.js

```diff
@@ -207,11 +224,16 @@ class CardStack extends React.Component<DefaultProps, Props, void> {
           if (header && header.left) {
             return header.left;
           }
-          const { renderLeftComponent } = this.props.headerComponent.defaultProps || {};
-          if (typeof renderLeftComponent === 'function') {
-            return renderLeftComponent(props);
+          if (props.scene.index === 0 || !props.onNavigateBack) {
+            return null;
           }
-          return null;
+          const prevSceneTitle = this._getHeaderTitleForScene(props.scenes[props.index - 1]);
+          return (
+            <HeaderBackButton
```

Comment:
> Maybe add scene.title

That would have to be `previousSceneTitle` kind of thing, which changes the API, right? An alternative if you really want to make the `Header` aware is to pass it `previousScene` as a prop and replace all checks for `props.scene.index === 0` with `!!previousScene`.